model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/records/UnknownField.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 Basis Technology Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.basistech.rosette.apimodel.recordsimilarity.records;
+
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class UnknownField implements RecordSimilarityField {
+    @JsonFormat(shape = JsonFormat.Shape.OBJECT)
+    JsonNode data;
+
+    @JsonCreator
+    public UnknownField(JsonNode data) {
+        this.data = data;
+    }
+
+    //There's probably a better way to do this
+    @JsonValue
+    public Object getData() {
+        if (data != null && data.isObject()) {
+            Iterator<Map.Entry<String, JsonNode>> fields = data.fields();
+            Map<String, JsonNode> map = new LinkedHashMap<>();
+            while (fields.hasNext()) {
```

Comment:
As for a better way to do this instead of putting the values into a map and reconverting it to Json - I couldn't find any, but Java stream interface could be used instead of an iterator and while loop to make this simpler.

Reply:
I've simplified the implementation for this method

Reply:
Looks good to me!